File: node/src/models/Record.ts
export class Record {
  id?: number;
  name: string;
  description?: string;
  createdDateTime?: string;
  created_date?: string;

  constructor(values?: Record) {
    this.id = values?.id;
    this.name = values?.name;
    this.description = values?.description;
    this.createdDateTime = values?.created_date ?? values?.createdDateTime;
  }
}

File: spring/src/main/java/com/record/persistence/entity/Record.java
package com.record.persistence.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.*;

import java.util.Date;

@Getter
@Entity
@Data
public class Record {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    private String name;
    private String description;
    private Date createdDate;

    public Record() {
        this.createdDate = new Date();
    }
}
